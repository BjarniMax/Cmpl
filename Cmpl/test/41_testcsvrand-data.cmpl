%data : a[i,i], c [ i ], i set
 
variables:
    x[1..dim(c[])]: real[0..];
    
objectives:
   c[]T * x[] ->max;
    
constraints:
   a[,] * x[] <= 100;
   echo "x "+  dim(x[]);
   echo "a "+ dim(a[1,]) + " " +dim(a[,1]);
     


