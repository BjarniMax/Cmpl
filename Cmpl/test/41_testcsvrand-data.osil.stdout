x 200
a 200 200
